### DISCUSSION TITLE
Would anyone else find this super useful? Custom URLs for non-theme extensions

### DISCUSSION THREAD

**Discussion by nbransby** [microsoft/vscode-discussions#173] [Extension Development QnA] [answered]
Would anyone else find this super useful?

If so, please upvote!

https://github.com/microsoft/vscode/issues/163593

Original issue and description here: https://github.com/microsoft/vscode/issues/139839

**Comment by digitarald** [comment, marked as answer]
Good luck. Will mark this as answered for now and recommend to keep the discussion going in https://github.com/microsoft/vscode/issues/163593 .
